src/sqlfluff/core/templaters/python.py

```diff
@@ -563,7 +563,7 @@ def _coalesce_types(elems: List[RawFileSlice]) -> str:
         types = {elem.slice_type for elem in elems}
         # Replace block types with templated
         for typ in list(types):
-            if typ.startswith("block_"):
+            if typ.startswith("block_"):  # pragma: no cover
```

Comment:
Here and elsewhere, we see that large areas of older slicing code are no longer used and could potentially be removed in a future PR.